lib/vagrant-mutate/mutate.rb

```diff
@@ -19,12 +19,17 @@ def execute
              'Specify version for input box') do |p|
           options[:version] = p
         end
+        # check for optional argument to force virtio disk driver in Vagrantfile (for libvirt)
+        o.on('--force-virtio',
+             'Force virtio disk driver in Vagrantfile for libvirt output provider') do |p|
+          options[:force_virtio] = true
+        end
       end
 
       argv = parse_options(opts)
       return unless argv
 
-      unless argv.length == 2
+      unless argv.length >= 2
```

Comment:
You shouldn't need to change this, right?. vagrant-mutate still takes exactly two arguments,  _box-name-or-file-or-url_ and _output-provider_.